infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/LRRollingUpgradeHandler.java

```diff
@@ -125,7 +129,8 @@ public boolean isLRUpgradeInProgress(TxnContext txnContext) {
      * @param txnContext All of the above must execute in the same transaction passed in.
      */
     public void migrateData(TxnContext txnContext) {
-        // Data migration to be added here.
+        // Currently only the LogReplicationMetadataManager needs data-migration
```

Comment:
I see, so here, you're migrating only the LRMetadata manager. But, at other place, different tables are involved in migration?

Reply:
from the analysis of V1 and V2 codepaths so far we only identified this layer needing schema migrations.
Did you encounter another layer which might need schema migrations? If so can you please share some details?